src/App.tsx

```diff
@@ -1,10 +1,25 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [keyPressed, setKeyPressed] = useState('');
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+  const detectKeyDown = (e: KeyboardEvent) => {
+    setKeyPressed(e.key);
+  };
+
+  useEffect(() => {
+    document.addEventListener('keydown', detectKeyDown, true);
```

Comment:
```suggestion
    document.addEventListener('keydown', detectKeyDown);
```

Reply:
You have the eyes of a hawk. And I thank you for that!